src/Http/Controllers/Configuration/SeatController.php

```diff
@@ -70,4 +74,26 @@ public function postUpdateSettings(SeatSettings $request)
             ->with('success', 'SeAT settings updated!');
     }
 
+    /**
+     * Help to fetch latest approved SDE version file from resources repository
+     *
+     * @return string Json
+     */
+    private function getApprovedSDE()
```

Comment:
This method is going to block the request and increase the load time. It should become an ajax request like its handled with api keys.